src/modules/Builder/pages/BuilderApplet/BuilderApplet.tsx

```diff
@@ -40,7 +54,7 @@ export const BuilderApplet = () => {
   }, [appletId]);
 
   return (
-    <>
+    <FormProvider {...methods}>
```

Comment:
pls add TODO to sync formData with session storage data on change form

Reply:
done